os/linux/appimage-builder.yml

```diff
@@ -0,0 +1,87 @@
+# appimage-builder recipe, see https://appimage-builder.readthedocs.io
+version: 1
+
+script: |
+    which unzip || apt-get update ; apt-get install wget unzip -y
```

Comment:
Guess unzip is no longer needed now :D Hihi :)

Reply:
indeed

Reply:
fixed